app/actions/accountsActions.js

```diff
@@ -26,42 +26,72 @@ const walletHasLabel = (wallet: Object, label: string) => {
 
 export const ID = 'accounts'
 
-export const updateAccountsActions = createActions(ID, (accounts: Array<Object>) => async (): Promise<Array<Object>> => {
-  const wallet = await getWallet()
-  const newWallet = { ...wallet, accounts }
-  await setStorage(STORAGE_KEY, newWallet)
-
-  return accounts
-})
+export const updateAccountsActions = createActions(
+  ID,
+  (accounts: Array<Object>) => async (): Promise<Array<Object>> => {
```

Comment:
Do you know the shape of the `Object` ? if so, describe it, same goes for the `Object` int he promise